upgrade_analysis/models/upgrade_analysis.py

```diff
@@ -31,6 +33,10 @@ class UpgradeAnalysis(models.Model):
     )
 
     log = fields.Text(readonly=True)
+    upgrade_path = fields.Char(
+        default=config.get("upgrade_path", False),
```

Comment:
:+1: for making it configurable.
(it is also more visible for the user).

Reply:
Yes, as I needed this to set the path from XMLRPC.